src/transform/WidenImage.js

```diff
@@ -12,7 +12,7 @@ const ancestorsToWiden = element => {
   while (el.parentNode) {
     el = el.parentNode
     // No need to walk above 'content_block'.
-    if (el.classList.contains('content_block')) {
+    if (el.classList && el.classList.contains('content_block')) {
```

Comment:
Can we change the loop to use `el.parentElement` instead? `classList` should always be defined for Elements but not necessarily Nodes.

https://developer.mozilla.org/en-US/docs/Web/API/Element
https://developer.mozilla.org/en-US/docs/Web/API/Node

Reply:
@joewalsh It looks like using `el.parentElement` on lines 12 and 13 fixes the problem. I'm looking at the save button console error message as part of the lazy loading fix. Would you be ok with changing this PR to just change lines 12 and 13?